src/lint/validators.nim

```diff
@@ -101,6 +108,16 @@ proc isString*(data: JsonNode; key: string; path: Path; context: string;
         result.setFalseAndPrint(&"Not a valid URL: {q s}", path)
     elif s.len > 0:
       if not isEmptyOrWhitespace(s):
+        if checkIsKebab:
+          if not isLowerKebab(s):
```

Comment:
The parameter is named `checkIsKebab` but the implementation is named `isLowerKebab`. I don't mind this much, but it is a tiny bit inconsistent.

Reply:
Renamed to `isKebabCase`.